game/screen_sing.cc

```diff
@@ -595,7 +594,7 @@ void ScreenSing::drawMenu() {
 	th.bg.draw();
 	// Loop through menu items
 	w = 0;
-	int player = 0;
+	std::size_t player = 0;
```

Comment:
this is actually a nice improvement, this way players cant go negative since a size_t is an unsigned 32 bit int.